app/controllers/katello/api/v2/content_views_controller.rb

```diff
@@ -145,21 +145,28 @@ def remove_from_environment
     param :system_environment_id, :number, :desc => N_("environment to reassign orphaned systems to")
     param :key_content_view_id, :number, :desc => N_("content view to reassign orphaned activation keys to")
     param :key_environment_id, :number, :desc => N_("environment to reassign orphaned activation keys to")
+    param :destroy_cv, :boolean, :desc => N_("delete the content view with all the versions and environments")
```

Comment:
i would avoid using' cv' as a parameter, and just use 'content_view'

Reply:
other than this small nitpick,t he rest of the ruby code looks good!

Reply:
@jlsherrill : Updated param to `destroy_content_view`